x-pack/plugins/aiops/server/routes/queries/fetch_index_info.ts

```diff
@@ -96,12 +95,7 @@ export const fetchIndexInfo = async (
   });
 
   const totalDocCount = (resp.hits.total as estypes.SearchTotalHits).value;
```

Comment:
Looks like the Typescript interface for search.hits.total is `total?: SearchTotalHits | long;`. Can we safely assume that resp.hits.total here will always be of type `SearchTotalHits`?

Reply:
Yes, the queries we use here use a hard coded `trackTotalHits: true` that's why we can safely do the `as estypes.SearchTotalHits`.